Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -92,10 +94,17 @@ - (instancetype)initWithEmailAddress:(NSString *)emailAddress logStore:(ARKLogSt
 
 - (void)composeBugReport;
 {
+    [self composeBugReportWithScreenshotLog:YES];
+}
+
+- (void)composeBugReportWithScreenshotLog:(BOOL)logScreenshot;
+{
     ARKCheckCondition(self.bugReportRecipientEmailAddress.length, , @"Attempting to compose a bug report without a recipient email address.");
     ARKCheckCondition(self.mutableLogStores.count > 0, , @"Attempting to compose a bug report without logs.");
     
-    if (!self.screenFlashView) {
+    self.logScreenshot = logScreenshot;
```

Comment:
I like your name better, the `logScreenshot` property reads a bit like a configuration of the bug reporter one might assume to be more permanent.